src/Compilers/CSharp/Portable/Symbols/Source/SourceSimpleParameterSymbol.cs

```diff
@@ -64,7 +64,7 @@ internal override bool IsExtensionMethodThis
 
         internal override bool IsMetadataIn
```

Comment:
>IsMetadataIn [](start = 31, length = 12)

It feels like we should adjust implementation of this property for SynthesizedParameterSymbolBase.